lib/typhoeus.rb

```diff
@@ -127,8 +127,6 @@ def self.before(&block)
   #     #=> :ok
   #   end
   #
-  # @param [ Block ] block The block to execute.
```

Comment:
Is it possible to use the the `yield` tag here instead? https://www.rubydoc.info/gems/yard/file/docs/Tags.md#yield

Reply:
Perhaps a `@yieldreturn` can explain what kind of Connection should be returned?  https://www.rubydoc.info/gems/yard/file/docs/Tags.md#yieldreturn